cocos/game/game.ts

```diff
@@ -811,7 +811,7 @@ export class Game extends EventTarget {
                 return this.onPostSubsystemInitDelegate.dispatch();
             })
             .then(() => {
-                console.log(`Cocos Creator v${VERSION}`);
+                console.log(`Here is Cocos Creator v${VERSION}, with you.`);
```

Comment:
Why ? And what do you want to express ?

Reply:
Just a more polite message printing, no other meanings.